pkg/cloudprovider/aws/instance.go

```diff
@@ -262,16 +261,11 @@ func (p *InstanceProvider) instanceToNode(ctx context.Context, instance *ec2.Ins
 				nodeName = aws.StringValue(instance.InstanceId)
 			}
 
+			// Since we no longer pre-bind, we need to ensure that all resources that were possibly considered for scheduling
+			// purposes are placed on the node.  The extended resources will be moved to an annotation, while the
+			// non-extended will be left on the node.
```

Comment:
I'm not entirely sure how pod-eni ever worked.  We weren't putting the resource on the node, but were just binding the pod. It should have been evicted by the kube-scheduler.